rln/src/public.rs

```diff
@@ -158,6 +176,9 @@ impl RLN<'_> {
         #[cfg(not(target_arch = "wasm32"))]
         let witness_calculator = circom_from_raw(circom_vec)?;
 
+        #[cfg(feature = "arkzkey")]
+        let proving_key = arkzkey_from_raw(&zkey_vec)?;
+        #[cfg(not(feature = "arkzkey"))]
```

Comment:
same as above, this arkzkey/zkey abstraction should not leak into the public module